src/Components/Coins/CoinsTab.css

```diff
@@ -1,6 +1,15 @@
 #coinsTab {
   width: 160px;
-  height: 80px;
+  height: 74px;
   background-color: blue;
   text-align: center;
 }
+
+#getMoreCoinsButton {
+  height: 31px;
+  width: 58px;
```

Comment:
why the weird exact numbers?

Reply:
I know what's the problem here, and yet decided it is not worth the time fixing, since it will be deleted soon.